.changeset/eighty-windows-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
```

Comment:
this is a minor change

Can we fix this without exposing a new css property? maybe with position: relative on the host element?

Reply:
1. this is only a problem in chrome

![image](https://github.com/user-attachments/assets/97cf96f7-3854-4056-8208-6a6f148cdc9c)

2. I think we can fix this with `position: relative; z-index: 1` on the container element, without requiring a minor release

![image](https://github.com/user-attachments/assets/6eac7036-14e6-4535-a91d-6ec0052dc1c1)